latest/examples/subspace/subspaceMultidim.json

```diff
@@ -0,0 +1,64 @@
+{
+    "coordinateSystems" : [
+        { "name " : "in", "axes" : [ {"name" : "0", "name" : "1", "name": "2", "name": "3", "name": "4" }] },
+        { "name " : "out", "axes" : [ {"name" : "x", "name" : "y", "name" : "z" }] }
+    ],
+    "coordinateTransformations" : [
+        { 
+            "type" : "sequence",
+            "name" : "5D-to-3D",
```

Comment:
Does this combination of `mapIndex` and `scale` really go from 5D data to 3D? And same for `5D-to-3D-not-contiguous` examples below?